examples/dir-extract.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import time
+import traceback
+from pathlib import Path
+
+from indico import IndicoClient
+from indico.config import IndicoConfig
+from indico.queries.documents import DocumentExtraction
+
+
+def ocr_files(target_files):
+
+    my_config = IndicoConfig(
+        host='dev.indico.io',
+        api_token_path=Path(__file__).parent / 'indico_api_token.txt'
```

Comment:
For the example I think we assume it's in their home dir?

Reply:
Yeah, I'm going to hold off on this example until wait on any complete job is done.